sherpa-onnx/csrc/keyword-spotter-transducer-impl.h

```diff
@@ -266,15 +266,25 @@ class KeywordSpotterTransducerImpl : public KeywordSpotterImpl {
   }
 
   void InitKeywords() {
-    // each line in keywords_file contains space-separated words
 
+#ifdef SHERPA_ONNX_ENABLE_WASM_KWS
+    // Due to the limitations of the wasm file system,
```

Comment:
Could you pass a keyword file from wasm?

We have been doing this for model files, such as `tokens.txt`.

Reply:
Sure, but  it needs to be recompiled when the keywords is modified, which is very inconvenient. Because token.txt and keywords.txt are both packaged into the sherpa-onnx-wasm-kws-main.data file, they cannot be modified and can only be recompiled to generate. What do you think?

Reply:
Ah, I see. Then please keep using your current approach.